index.yaml

```diff
@@ -306,13 +306,13 @@ indexes:
   - name: commit_message
   - name: commit_type
 
-- kind: UserContributionScoringModel
+- kind: UserContributionProficiencyModel
   properties:
   - name: deleted
   - name: user_id
   - name: score
 
-- kind: UserContributionScoringModel
+- kind: UserContributionProficiencyModel
```

Comment:
Considering this kinda issue never gets caught until we deploy the release branch on the server, @seanlip Do you think we should have a check to make sure this doesn't happen in the first place? (Maybe a backend test (that way we will be able to fetch list of all the model class name, but I'm not sure whether backend test is a good place for this check.)

Reply:
Hi @DubeySandeep, I'm OK with that. Perhaps file an issue explaining clearly the desired proposal?

Reply:
As per offline discussion with @seanlip, we realized that this kinda issue has never happened before (because we don't change Model's name anytime) and we don't expect similar changes in the future, so having a check for the index.yaml file isn't that required. (We can have this check if this issue happens again in the future!)